model/apmevent.go

```diff
@@ -91,11 +91,11 @@ func (e *APMEvent) MarshalJSON() ([]byte, error) {
 // MarshalFastJSON marshals e as JSON, writing the result to w.
 func (e *APMEvent) MarshalFastJSON(w *fastjson.Writer) error {
 	var outProto modelpb.APMEvent
-	e.toModelProtobuf(&outProto)
+	e.ToModelProtobuf(&outProto)
 	return outProto.MarshalFastJSON(w)
 }
 
-func (e *APMEvent) toModelProtobuf(out *modelpb.APMEvent) {
+func (e *APMEvent) ToModelProtobuf(out *modelpb.APMEvent) {
```

Comment:
Shouldn't there be a comment? (I'm a bit surprised the linter doesn't require one)

Reply:
I guess we could add it. Keep in mind, this is temporary and it's public just to make it easier to migrate. The `model` package will go away along with this method.